src/api/registerer.ts

```diff
@@ -22,10 +22,12 @@ import { UserAgent } from "./user-agent";
  */
 export class Registerer {
   private static readonly defaultExpires = 600;
+  private static readonly defaultRefreshFrequency = 50;
```

Comment:
Maybe default to 99? For a 1 hour expiration this would be about 36 seconds before expiration.